TypeError: context.succeed is not a function

Hello All --

I was wondering if you could help me with the error I am running into. I've used lambda-cloudwatch-slack before on another AWS account. Now I need to install it again on my new AWS account. Do you know where I am going wrong? 

processing cloudwatch notification
message posted successfully
/home/ec2-user/lambda-cloudwatch-slack/index.js:299
      context.succeed();
              ^

TypeError: context.succeed is not a function
    at /home/ec2-user/lambda-cloudwatch-slack/index.js:299:15
    at IncomingMessage.<anonymous> (/home/ec2-user/lambda-cloudwatch-slack/index.js:38:9)
    at emitNone (events.js:72:20)
    at IncomingMessage.emit (events.js:166:7)
    at endReadableNT (_stream_readable.js:913:12)
    at nextTickCallbackWith2Args (node.js:442:9)
    at process._tickDomainCallback (node.js:397:17)
make: *** [test] Error 1

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "lambda-cloudwatch-slack",
-  "version": "0.1.0",
+  "version": "0.1.2",
   "description": "Better Slack notifications for AWS CloudWatch",
   "authors": [
     "Assertible <admin@assertible.com>",
@@ -17,7 +17,7 @@
     "url": "^0.11.0"
   },
   "devDependencies": {
-    "node-lambda": "^0.8.6"
+    "node-lambda": "0.8.11"
   },
   "keywords": [
     "aws",

